src/wasm/WasmModule.cpp

```diff
@@ -199,8 +252,31 @@ void WasmModule::restore(const std::string& snapshotKey)
     // Map the snapshot into memory
     uint8_t* memoryBase = getMemoryBase();
     reg.mapSnapshot(snapshotKey, memoryBase);
+}
+
+void WasmModule::ignoreThreadStacksInSnapshot(const std::string& snapKey)
+{
+    std::shared_ptr<faabric::util::SnapshotData> snap =
+      faabric::snapshot::getSnapshotRegistry().getSnapshot(snapKey);
+
+    uint32_t threadStackRegionStart =
```

Comment:
Could we maybe add a comment here on the stack structure, or point somewhere where this structure may be; justifying in the process why we add `1`.